web/src/routes/admin/user-management/+page.svelte

```diff
@@ -220,7 +221,7 @@
                       </button>
                     {/if}
                   {/if}
-                  {#if isDeleted(immichUser)}
+                  {#if isDeleted(immichUser) && immichUser.status != "removing"}
```

Comment:
Looking to see if there is a way to reference the enum at https://github.com/immich-app/immich/pull/7681/files#diff-5769839dae446ca9c68221d6d1ec556da0810925a90ffc10096e8061d2ff3a3bR26 or if I need to recreate it in web